readme.md

```diff
@@ -105,23 +113,16 @@ To set up your database and load initial data, run the following commands:
 
 Log into the admin with the credentials ``admin / changeme``.
 
-### Note on demo search:
-
-Because we can't (easily) use ElasticSearch for this demo, we use wagtail's native DB search.
-However, native DB search can't search specific fields in our models on a generalized `Page` query.
-So for demo purposes ONLY, we hard-code the model names we want to search into `search.views`, which is
-not ideal. In production, use ElasticSearch and a simplified search query, per
-[http://docs.wagtail.io/en/v1.8.1/topics/search/searching.html](http://docs.wagtail.io/en/v1.8.1/topics/search/searching.html).
-
-### Heroku deployment:
+Deploy to Heroku
+----------------
 
-If you need to deploy your demo site to a publicly accessible server [Heroku](https://heroku.com)
-provides a one-click deployment solution:
+If you don't want to test locally you can deploy a demo site to a publicly accessible server with [Heroku's](https://heroku.com)
+one-click deployment solution to their free 'Hobby' tier:
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/wagtail/bakerydemo)
 
 If you do not have a Heroku account, clicking the above button will walk you through the steps
-to generate one.  After which, you will be presented with a screen to configure your app. For our purposes,
+to generate one.  At this point you will be presented with a screen to configure your app. For our purposes,
 we will accept all of the defaults and click `Deploy`.  The status of the deployment will dynamically
 update in the browser. Once finished, click `View` to see the public site.
 
```

Comment:
To bring up an initial build on Heroku the user doesn't need to have S3 set-up, it's only if they wanted it to persist that they'd need to do that. I have to say it's a pretty seamless deployment (I was surprised dealing with it yesterday for the image issue just how straightforward it is) and is certainly something I could imagine suggesting a client do if they were interested in exploring Wagtail. I don't think we can presume that people will always want, or be able, to deploy locally and Heroku is currently our best option. Hopefully we can offer others in the future though.

On the note around starting projects from our codebase, I think Heroku would make this more difficult. The deployment is tied to our git repo, so at the very least, if the user would have to fork the repo before they started using it for their own thing, which hopefully would encourage them to not do that and start from scratch based on their own needs.